index.bs

```diff
@@ -1300,10 +1300,12 @@ lt="ReadableStreamDefaultReader(stream)">new ReadableStreamDefaultReader(<var>st
 <h5 id="default-reader-closed" attribute for="ReadableStreamDefaultReader" lt="closed">get closed</h5>
 
 <div class="note">
-  The <code>closed</code> getter returns a promise that will be fulfilled when the stream becomes closed or the
-  reader's lock is <a lt="release a read lock">released</a>, or rejected if the stream ever errors.
+  The <code>closed</code> getter returns a promise that will be fulfilled when the stream becomes closed, or rejected if
+  the stream ever errors or the reader's lock is <a lt="release a read lock">released</a> before the stream finishes
+  closing or the stream errors.
```

Comment:
> or the stream errors.

I don't think you need to say this, as you've already made it clear that the promise rejects on error.

Reply:
lol